tests/test_models.py

```diff
@@ -82,6 +84,8 @@ def test_ops_layout_transparent():
 def test_networks():
     NETWORK_LIST = (
         'mobilenet_v1_0.25_128',
+        'object_detection_3d_sneakers',
```

Comment:
In most cases, I'd like to keep the `master` branch small enough. That means, we keep only test purpose operator TFLite model on `master` branch, and track more network-wide models in [`more-model-test`](https://github.com/jackwish/tflite2onnx/tree/more-model-test) branch. I can help to move this model to that branch after this PR got merged.